CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.18.0)
+
+project(deplex LANGUAGES CXX DESCRIPTION "Library for Depth Plane Extraction")
+
+option(BUILD_EXAMPLES "Build C++ examples of deplex usage" ON)
+option(BUILD_PYTHON "Build Python bindings" ON)
+
+add_subdirectory(3rd_party)
+add_subdirectory(cpp)
```

Comment:
It is usually named `src`

Reply:
https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs